wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -0,0 +1,208 @@
+    - Declaring the match as forfeited, or as a win by default for the other team.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future GBC series.
+    - Disqualification from the current and future official & commnunity tournaments, until appealed.
+21. The Host reserves the right to modify these rules at any moment. Any such canges will be announced in advance.
+
+### Tournament registration
+
+1. Please send the Host **a private message** and **fill [this form](https://wj.qq.com/s2/9902248/95cf/)** to register. 
+2. Each player could send a VOD clips or an in-game replays to Host, which will be used on Draft phase. 
+3. To ensure valid and serious registrations, every registered user will be manually checked by the Host.
+   - To be successfully registered, a player must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
+   - **No rank limitations for this tournament.** host have the right to choose several best players to participate in showmatch (showcase by a match) **instead of** the actual tournament for balance issues.
+4. A list of all successfully registered players will be published after the Registration Phase.
+   - All valid registered players will have the chance to pricipate Draft phase.
+5. Organizers, Mappool Selectors and referees must not participate as a player in the tournament.
+6. Organizers keeps right to decline any suspend account pricipate this tournament.
+
+### Draft phase instructions
+
+1. Host will declare one captain for each team on 24 hours before Draft phase.
+   - drafting order will also be declared by Host.
+   - Captains are selected by player's performance and preference.
+   - Other valid players not belongs to captains , will automaticly joined into a group, wating to join a team, herein called *player pool*.
+2. Captains can decided their team members on draft phase.
+   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms. 
+   - On draft phase, Captains will also allowed to see a VOD clips or an in-game replays provided by potential players, if possible.
+3. An initial 120-second planning phase begins, during which allowed first captain to thinking which player should be picked first.
+4. The picking phase begins afterward. The pick order consists of 2 phases that repeat twice, and every captains gets to pick 4 times.
+   - 1st , 3rd phase: 1st to 16th captain, increasing order.
+   - 2nd , 4th phase: 16th to 1st captain, decreasing order.
+5. Each Captains has 60 seconds to pick an player from the player pool on their turn. 
+   - The picked player will join into the captain's team, then removed from the player pool.
+   - If the captain does not pick an player within 60 seconds, a random player from the remaining player pool will be assigned.
+   - The captain are only allowed to pick one player on their turn.
```

Comment:
i think it might not bad to keep the third sub rules, may add "only" better ? (`- The captain are only allowed to pick only one player on their turn.`)

Reply:
can instead emphasise the "one" in the rule itself (`Each captain has 60 seconds to pick **one player** from the player pool on their turn. `), which already makes the third subrule redundant

and the first subrule is redundant since there's no way to interpret rule 5 in any other way

Reply:
reasonable.